src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
Yeah, `RPP_NO_DEFAULTS ` is more generic, +1 for the name.

It's still pretty intrusive in our system in that we need to modify some package to set this env variable when being sourced (I was thinking in `roslib`) and then you need to disable that when you're packaging the same package for Debian. For newer ROS distributions that should be ok, but it is not going to be possible to introduce it in EOL ROS distros, e.g. Groovy or Hydro, but that should be fine since this doesn't really show up until Xenial, i.e. there shouldn't be ROS packages in `/usr` before that so adding it to the RPP by default will at worst slowdown searches for package manifests.

I think that sounds like the best compromise that I've heard. @dirk-thomas what do you think about having `/usr` in by default and having a way to disable those defaults by env variable?